client/components/telecom/telephony/abbreviatedNumbers/telecom-telephony-abbreviated-numbers.modal.html

```diff
@@ -1,171 +1,71 @@
-                data-ui-validate="{available: 'true'}"
-                required
-                data-ng-readonly="AbbreviatedNumberModal.readOnly.abbreviatedNumber"
-                data-ng-model="AbbreviatedNumberModal.data.abbreviatedNumber" />
-            <span class="help-block"
-                  role="alert"
-                  data-ng-if="abbreviatedNumberForm.abbreviatedNumber.$touched"
-                  data-ng-messages="abbreviatedNumberForm.abbreviatedNumber.$error">
-                <span data-ng-message="pattern"
-                      data-ng-bind="AbbreviatedNumberModal.pattern.errorMessage">
-                </span>
-                <span data-ng-message="available"
-                      data-translate="telephony_abbreviated_numbers_not_free_error">
-                </span>
-            </span>
+        <div class="text-center" data-ng-if="AbbreviatedNumberModal.loading.updating">
+            <oui-spinner></oui-spinner>
         </div>
 
-        <!-- Number -->
-        <div class="form-group"
-             data-ng-class="{
-                'has-error': abbreviatedNumberForm.destinationNumber.$touched && abbreviatedNumberForm.destinationNumber.$invalid
-             }">
-            <label for="destinationNumber"
-                   class="control-label"
-                   data-translate="telephony_abbreviated_numbers_number">
-            </label>
-            <input
-                type="text"
-                name="destinationNumber"
-                id="destinationNumber"
-                class="form-control"
-                data-translate
-                translate-attr-placeholder="telephony_abbreviated_numbers_number"
-                data-ng-pattern="AbbreviatedNumberModal.numberPattern"
-                required
-                data-ng-readonly="AbbreviatedNumberModal.readOnly.destinationNumber"
-                data-ng-model="AbbreviatedNumberModal.data.destinationNumber"/>
-            <span class="help-block"
-                  role="alert"
-                  data-ng-if="abbreviatedNumberForm.destinationNumber.$touched"
-                  data-ng-messages="abbreviatedNumberForm.destinationNumber.$error">
-                <span data-ng-message="pattern"
-                      data-translate="telephony_abbreviated_numbers_number_error">
-                </span>
-            </span>
-        </div>
+        <div data-ng-if="!AbbreviatedNumberModal.loading.updating">
+            <oui-field label="{{:: 'telephony_abbreviated_numbers_id' | translate }}"
+                       error-messages="{pattern: AbbreviatedNumberModal.pattern.errorMessage}">
+                <input
+                    type="text"
+                    name="abbreviatedNumber"
+                    id="abbreviatedNumber"
+                    class="oui-input"
+                    data-ng-pattern="AbbreviatedNumberModal.pattern.regexp"
+                    data-ui-validate="{available: 'true'}"
```

Comment:
Message is missing for this validation and field component will not support it.
@neolitec @JDownloader

Reply:
It's removed, all api errors are displayed in a oui-message not in a field